<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountAuthorization</class>
 <widget class="QWidget" name="AccountAuthorization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>628</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>账号授权</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>411</width>
     <height>311</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnOk">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>103</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>确认注册</string>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/image/update.png</normaloff>:/image/update.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="UserName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>205</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>请输入用户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UserRePwd">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>205</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>请再次输入密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>103</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string notr="true">重置</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/image/delete.png</normaloff>:/image/delete.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="labUserPwd">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>112</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labUserPwd_2">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>162</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>确认密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labUserName">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UserPwd">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>205</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>请输入密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>62</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>账号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UserNum">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>205</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>请输入你注册的账号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>212</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择身份：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>100</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>店员</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>店长</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>临时工</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
